script/install-stack.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+if [ -f "${HOME}/.local/bin/stack" ]
+then
+    echo 'Stack is already installed.'
+else
+    echo "Installing Stack for ${TRAVIS_OS_NAME}..."
+    URL="https://www.stackage.org/stack/${TRAVIS_OS_NAME}-x86_64"
```

Comment:
Maybe the os name should be passed as an argument to the script so that this isn't travis-specific?